src/strategies/marsecosystem/README.md

```diff
@@ -0,0 +1,67 @@
+# marsecosystem
+
+This is the most common strategy, it returns the balances of the voters for a specific ERC20 token.
+
+Here is an example of parameters:
+
+```json
+{
+  "token": "0x7859B01BbF675d67Da8cD128a50D155cd881B576",
+  "miningMasters": [
+    { "address": "0xc7B8285a9E099e8c21CA5516D23348D8dBADdE4a", "pid": 0 },
+    { "address": "0x48C42579D98Aa768cde893F8214371ed607CABE3", "pid": 0 }
+  ],
+  "upMiningMasters": [
```

Comment:
Can we add limits to these arrays? Accepting unlimited values could cause memory issues

Reply:
thanks, i have added the limit in version [cc70113](https://github.com/snapshot-labs/snapshot-strategies/pull/937/commits/cc70113ea321834bf45c515b711bb73c50ac8331)